addons/refuel/functions/fnc_connectNozzleAction.sqf

```diff
@@ -0,0 +1,109 @@
+/*
+ * Author: GitHawk et.al
+ * Calculates a connection for tilting
+ * With code from ace_attach
```

Comment:
Is it copy pasted part of the code or entire function? If it's entire function I see no reason not to add ace_attach as a requirement for this module.